vnext/ReactUWP/Views/FlyoutViewManager.cpp

```diff
@@ -249,9 +248,11 @@ void FlyoutShadowNode::createView() {
 }
 
 void FlyoutShadowNode::onDropViewInstance() {
-  m_isOpen = false;
-  m_flyout.Hide();
-  s_cOpenFlyouts -= 1;
+  if (m_isOpen) {
+    m_isOpen = false;
+    m_flyout.Hide();
+    s_cOpenFlyouts -= 1;
```

Comment:
My understanding is that each reactControl instance will be on a separate thread, so we made  s_cOpenFlyouts thread_local.

Reply:
I don't think so. maybe @acoates-ms knows about it

Reply:
Worst case is that we'll have a total count of open flyouts for all reactControl instances rather than a count for each instance. This will result in slightly larger shadows for the overlapping flyouts.

Reply:
...which I think is tolerable...or at least a big improvement over the funky shadows :-)